sdks/go/examples/build.gradle

```diff
@@ -77,3 +77,10 @@ golang {
     continueOnFailure = true
   }
 }
+
+// Run this task to validate the Go environment setup for contributors
```

Comment:
No need to add these any more since they're added in https://github.com/apache/beam/pull/14467/files ?

Reply:
True. I merged the master (which already has these changes) with this branch to remove this from my PR.